wagtail/wagtailadmin/static_src/wagtailadmin/js/page-chooser.js

```diff
@@ -1,4 +1,4 @@
-function createPageChooser(id, pageTypes, openAtParentId, canChooseRoot) {
+function createPageChooser(id, pageTypes, openAtParentId, canChooseRoot, UserPerms) {
```

Comment:
This variable should be `userPerms` (with a small `u`) - capitalised identifiers are only for classes / constructors.